src/fondant/runner.py

```diff
@@ -85,3 +85,36 @@ def get_name_from_spec(self, input_spec: str):
         with open(input_spec) as f:
             spec = yaml.safe_load(f)
             return spec["pipelineInfo"]["name"]
+
+
+class VertexRunner(Runner):
+    def __resolve_imports(self):
+        import google.cloud.aiplatform as aip
+
+        self.aip = aip
+
+    def __init__(self, project_id: str, project_region: str, service_account: str = ""):
```

Comment:
```suggestion
    def __init__(self, project_id: str, project_region: str, service_account: t.Optional[str]=None):
```

https://github.com/googleapis/python-aiplatform/blob/0fa47aea08b635159e9c6dc64f7d330f047dfb5d/google/cloud/aiplatform/preview/jobs.py#L232